libcrux_crypto/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "openmls_libcrux_crypto"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+getrandom = "0.2.12"
+libcrux = { git = "https://github.com/cryspen/libcrux", features = ["rand"] }
```

Comment:
Any chance we can pin this on a version?

Reply:
I pinned it to a commit; we don't have a recent release yet